microcloud/test/includes/microcloud.sh

```diff
@@ -102,8 +102,18 @@ set_debug_binaries() {
 
   if [ -n "${MICROCLOUD_DEBUG_PATH}" ] && [ -n "${MICROCLOUDD_DEBUG_PATH}" ]; then
     echo "==> Add debug binaries for MicroCloud."
-    lxc exec "${name}" -- rm -f /var/snap/microcloud/common/microcloudd.debug
-    lxc exec "${name}" -- rm -f /var/snap/microcloud/common/microcloud.debug
+
+    # lxc exec may lock up on fast-running commands if running in the background, so we need to sleep 1s.
+    lxc exec "${name}" -- sh -c "
+      sleep 1
+      if test -e /var/snap/microcloud/common/microcloudd.debug ; then
+        rm -f /var/snap/microcloud/common/microcloudd.debug
+      fi
+
+      if test -e /var/snap/microcloud/common/microcloud.debug ; then
+        rm -f /var/snap/microcloud/common/microcloud.debug
+      fi
+    "
```

Comment:
I'd prefer `sleep 1` to be out of that bit and then simply do `lxc exec "${name}" -- rm -f /var/snap/microcloud/common/microcloudd.debug /var/snap/microcloud/common/microcloud.debug`